ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/hooks/useVulnerabilityState.ts

```diff
@@ -0,0 +1,11 @@
+import useFeatureFlags from 'hooks/useFeatureFlags';
+import useURLStringUnion from 'hooks/useURLStringUnion';
+import { vulnerabilityStates } from 'types/cve.proto';
+
+export default function useVulnerabilityState() {
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+    const isUnifiedDeferralsEnabled = isFeatureFlagEnabled('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL');
+    const [vulnerabilityStateKey] = useURLStringUnion('vulnerabilityState', vulnerabilityStates);
```

Comment:
Once the feature flag is removed, we can extract this hook and use the `useURLStringUnion` call inline in its place.